src/portfft/descriptor.hpp

```diff
@@ -637,6 +637,7 @@ class committed_descriptor {
         std::vector<sycl::kernel_id> ids;
         auto in_bundle = sycl::get_kernel_bundle<sycl::bundle_state::input>(queue.get_context(),
                                                                             detail::get_transpose_kernel_ids<Scalar>());
+        in_bundle.template set_specialization_constant<detail::SpecConstComplexStorage>(params.complex_storage);
```

Comment:
No need to `SpecConstComplexStorage` here as its already being set [here](https://github.com/codeplaysoftware/portFFT/blob/0e8aed26527fbf22ffc9273753c333045c2adf76/src/portfft/descriptor.hpp#L449) .